tests/integration/security/remote_jwks/remote_jwks_test.go

```diff
@@ -71,6 +73,20 @@ func TestRemoteJwks(t *testing.T) {
 							}))
 					},
 				},
+				{
+					name:       "remote-jwks-with-service-entry",
+					policyFile: "./testdata/requestauthn-with-se-timeout.yaml.tmpl",
+					timeout:    "20ms",
+					delay:      "2s",
```

Comment:
why do we need this delay?

Reply:
they delay is needed to guarantee the failure but I have to say that in this scenario it is too high.
Changed to timeout 10ms and delay 30ms